src/Test/Unit/Process/Deploy/InstallUpdate/ConfigUpdate/UrlsTest.php

```diff
@@ -83,43 +99,42 @@ public function testExecute()
         $this->process->execute();
     }
 
-    /**
-     * @param bool $envIsMasterBranchWillReturn
-     * @param InvokedCount $envIsUpdateUrlsEnabledExpects
-     * @dataProvider executeSkippedDataProvider
-     */
-    public function testExecuteSkipped(
-        bool $envIsMasterBranchWillReturn,
-        InvokedCount $envIsUpdateUrlsEnabledExpects
-    ) {
+    public function testExecuteSkippedIsMasterBranch()
+    {
         $this->environmentMock->expects($this->once())
             ->method('isMasterBranch')
-            ->willReturn($envIsMasterBranchWillReturn);
-        $this->stageConfigMock->expects($envIsUpdateUrlsEnabledExpects)
+            ->willReturn(true);
+        $this->stageConfigMock->expects($this->once())
             ->method('get')
-            ->with(DeployInterface::VAR_UPDATE_URLS)
+            ->with(DeployInterface::VAR_FORCE_UPDATE_URLS)
             ->willReturn(false);
         $this->loggerMock->expects($this->once())
             ->method('info')
-            ->with('Skipping URL updates');
+            ->with($this->stringContains('Skipping URL updates because we are deploying to a Production or Staging'));
```

Comment:
```suggestion
            ->with($this->stringContains('Skipping URL updates because we are deploying to a Production or Staging environment.'));
```